tests/crowdsourcing/tasks/qa_data_collection/test_qa_data_collection/test_base_task.yml

```diff
@@ -0,0 +1,31 @@
+- inputs: {}
```

Comment:
great! But we can remove `expected_states/` now because we have this file instead